eng/build.yml

```diff
@@ -26,7 +26,7 @@ phases:
     steps:
     - ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
       - script: build.cmd
-                  -test -sign -pack -publish
```

Comment:
You should not have to do this. if the right properties are set, arcade will do the right thing already.